### ISSUE TITLE
Renamed items showing in kill messages.

### ISSUE THREAD

**Issue by walkersgaming** [CivMC/Civ#278]
Back when I made [this PR](https://github.com/CivMC/CivChat2/pull/14) to improve global kill messages to buff the use of Wordbank, it was agreed that renamed items shouldn't show in the kill message, only Wordbanked item names.

However, despite working when tested on Live, some sword names still show in global kill messages. Guessing there's a different NBT data structure causing the issue.

**Comment by RedDevel2** [comment]
Can confirm it's inconsistent with how it sometimes does and sometimes doesn't use the item name.

**Comment by okx-code** [comment]
It's only broken on admin-spawned items that contain colours in their name, it works fine for anvil renamed items

**Comment by RedDevel2** [comment]
No, It's also for renamed ones. It'll show up dark gray same as rest of the text.

**Comment by Diet-Cola** [comment]
Uhhh is this worth holding off fixing until 1.20.5 stuff in the dev chat?
